roles/matrix-bridge-appservice-slack/templates/systemd/matrix-appservice-slack.service.j2

```diff
@@ -0,0 +1,37 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix Appservice Slack server
+{% for service in matrix_appservice_slack_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_appservice_slack_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+
+[Service]
+Type=simple
+ExecStartPre=-/usr/bin/docker kill matrix-appservice-slack
+ExecStartPre=-/usr/bin/docker rm matrix-appservice-slack
+
+# Intentional delay, so that the homeserver (we likely depend on) can manage to start.
+ExecStartPre=/bin/sleep 5
+
+ExecStart=/usr/bin/docker run --rm --name matrix-appservice-slack \
+			--log-driver=none \
+			--network={{ matrix_docker_network }} \
+			-v {{ matrix_appservice_slack_base_path }}:/usr/src/app/userconfig:z \
```

Comment:
This likely needs to be adjusted, so that it mentions the 2 new variables: `matrix_appservice_slack_config_path` and `matrix_appservice_slack_data_path`.

Maybe it should be like this:

```
-v {{ matrix_appservice_slack_config_path }}:/usr/src/app/userconfig:z \
-v {{ matrix_appservice_slack_data_path }}:/data:z \
```

As for the data path, I don't see the bridge referring to any "data" or "database" directory or path, so I'm not sure how we'll use it at all. Maybe there's some hidden configuration setting or something.

In any case, it does sound better to at least make an attempt at separating `./config` from `./data`, even if we can't use it now and all files end up in `./config`.